Documentation/Usage.md

```diff
@@ -146,81 +146,116 @@ following files from the libgit2 system-package are of interest:
     /usr/local/lib/libgit2.dylib      # .so on Linux
     /usr/local/include/git2.h
 
-Swift packages that provide module maps for system libraries are handled
-differently from regular Swift packages.
+**Note:** the system library may be located elsewhere on your system, such as
```

Comment:
I think that we should clear up some of the text here.  Windows is a support platform, and these paths are not relevant there.  Additionally, `pkg-config` is not a commonly available utility on Windows.

Reply:
Added an entry on Windows paths.